packages/next/src/server/render.tsx

```diff
@@ -1536,12 +1537,20 @@ export async function renderToHTMLImpl(
   return new RenderResult(optimizedHtml, renderResultMeta)
 }
 
-export async function renderToHTML(
+export type PagesRender = (
   req: IncomingMessage,
   res: ServerResponse,
   pathname: string,
   query: NextParsedUrlQuery,
   renderOpts: RenderOpts
-): Promise<RenderResult> {
+) => Promise<RenderResult>
+
+export const renderToHTML: PagesRender = (
+  req,
+  res,
+  pathname,
+  query,
+  renderOpts
+): Promise<RenderResult> => {
```

Comment:
are we able to skip this return type as it's defined above?
```suggestion
) => {
```